oat-sa/extension-tao-testqti
# Fix/uno 196/item permissions

Pull request summary
 
Related to : https://oat-sa.atlassian.net/browse/UNO-196
 
1. Deleted old filtration that was done after the resource selection (thats why we didn't have a results even if they are there).
2. Permissions added to the return data
 
#### Steps to reproduce  (for bugs)
- Log in as a Author
- Go to test page
- Create test and go to authoring tab
- Click “View resources as a list“ button
- Observe the screen
  
#### How to test
 
Install all dependencies. After that all resources will be shown with permission-markers (lock icon on the resource).
 
Requires :
 - [ ] https://github.com/oat-sa/tao-core/pull/2705
